### PR TITLE
 #4 Use explicit dependencies

### LINKED ISSUE #4: explicit dependencies
Details [here](https://ci.linagora.com/linagora/lgs/openpaas/esn/-/wikis/Frontend-Split/Stage-1/Explicit-Dependencies)

### PR DESCRIPTION
**Pull request by MichaelBailly**
Resolves #4

### CODE REVIEW

File: src/app/mailto.js [hunk 1]
```diff
@@ -1,71 +1,89 @@
...
+  'esn.escape-html',
+  'esn.async-action',
+  'esn.user',
+  'esn.session',
+  'esn.escape-html',
+  'esn.registry',
+  'esn.module-registry',
+  'esn.user-configuration',
+  'esn.datetime',
+  'esn.i18n',
```
Comment (rezk2ll) [inline src/app/mailto.js:51]: those modules were not required on purpose? others below and up ( 21/46 dependencies were required )

Comment (MichaelBailly) [reply src/app/mailto.js:51]: I don't understand your question...

FYI what I did here: 
* remove the IIFE
* remove lodash.wrapper dependency
* add the require('...') (explicit dependency)

This PR is more readable if you use Split diff view and "Hide whitespaces changes".

Can you rephrase your question ?

Comment (rezk2ll) [reply src/app/mailto.js:51]: my question was: some modules like `esn.i18n` were not required. like:

```js
require('esn-frontend-common-libs/src/frontend/js/modules/i18n/i18n.module.js');
```

is it intentional?

Comment (MichaelBailly) [reply src/app/mailto.js:51]: OK I got it sorry ! What I did is start the app and require the missing angular dependancies until AngularJS is happy to bootstrap the app. I agree that it's too fragile, I'll add the missing `require`
